server/src/modules/search/search.types.ts

```diff
@@ -0,0 +1,8 @@
+export type SearchEntry = {
+  postId: number
+  title?: string
+  description?: string | null
+  answers?: string[]
+  agencyId?: number
+  topicId?: number | null
+}
```

Comment:
Should this belong to `shared/`?

Reply:
Hm yes, it would come it handy when the query is hooked up to frontend. Will add it in `shared/` then, thanks!